Docker/Dockerfile

```diff
@@ -22,8 +22,12 @@ RUN mkdir -p /data/srv/tmp && mkdir -p /data/srv/Publisher && mkdir -p /data/srv
 WORKDIR ${WDIR}
 
 # install
-COPY --chown=${USER}:${USER} install.sh monitor.sh ./
+COPY --chown=${USER}:${USER} install.sh addGH.sh monitor.sh ./
 RUN echo "Y" > yes.repo && sh install.sh < yes.repo
+RUN rm yes.repo
```

Comment:
each docker `RUN` command creates an additional layer, i.e. increases image size (other examples of commands that creates layers are `ADD, COPY`). So, since `rm yes.repo` is related to the installation that happens in line 26, I would change it to be like this:
`RUN echo "Y" > yes.repo && sh install.sh < yes.repo && rm yes.repo` and delete line 27.
this way we create only one layer instead of two and do not increase the size of the build. 

we could also add `RUN ./addGH.sh` using `&&` to line 26 to avoid adding additional layer, but this is more matter of taste given that `./addGH.sh` does unrelated thing to what is happening in line 26

Reply:
thanks for those insights on layering !

Reply:
all in all I prefer to leave the install.sh in its own layer.

Reply:
> all in all I prefer to leave the install.sh in its own layer.

I'd do the same